src/implementations/twig/components/link/link.html.twig

```diff
@@ -77,10 +79,11 @@
   {% set _link = _link|merge(link) %}
 {% endif %}
 
-{% if _link.external and _link.icon_path is not empty %}
+{% if _link.external and (_link.icon_path is not empty or _link.icon_wt_markup) %}
```

Comment:
this is one of those cases where i don't understand why adding the parameter at the component level, we merge the data provided by the user, why do we need to add this complexity and additional processing with also the consequence of forcing all the icons to follow the same approach?